generic/cleanerplugin/default.xml

```diff
@@ -600,7 +600,7 @@ QLineEdit#le_status_text {
                     <item type="QKeySequence">Alt+H</item>
                 </hide>
                 <history type="QKeySequence" comment="Open the message history">Ctrl+H</history>
-                <user-info type="QKeySequence" comment="Show the user info of the contact">Ctrl+I</user-info>
```

Comment:
Both versions sound strange.
Maybe "Show contact's vCard" would be good enough.
Overall it's strange to have this bunch or articles in the hotkeys settings. For example when we talk about contact's vCard we basically make our accent on the "vcard" word and consider "contact" as rather an adjective which hardly needs an article. Having an article for "vCard" sounds even more weird.
And if we go to the original "of the contact". There are two ways to think about it. 1) To assume a specific roster contact. 2) To talk about contacts in general.  These two ways assume different articles. So this phrases rather brings some confusion.

I'm not a linguist but I have this feeling in my guts.